src/transformation/transformers/class/new.ts

```diff
@@ -0,0 +1,64 @@
+import * as ts from "typescript";
+import * as tstl from "../../../LuaAST";
+import { FunctionVisitor, TransformationContext } from "../../context";
+import { DecoratorKind, getCustomDecorators } from "../../utils/decorators";
+import { InvalidDecoratorArgumentNumber, InvalidNewExpressionOnExtension } from "../../utils/errors";
+import { importLuaLibFeature, LuaLibFeature } from "../../utils/lualib";
+import { transformArguments } from "../call";
+
```

Comment:
I think it's a little strange we have both class/creation and class/new, why are they not the same thing?

Reply:
`creation` contains the `createClassCreationMethods` function, which creates some common class setup boilerplate. Any suggestions for a better name?